packages/wallets/src/features/cfd/screens/MT5TradeScreen/MT5TradeLink/MT5MobileRedirectOption.tsx

```diff
@@ -0,0 +1,74 @@
+/* eslint-disable camelcase */
+import React, { FC } from 'react';
+import { THooks } from 'src/types';
+import {
+    LabelPairedChevronRightLgFillIcon,
+    StandaloneLaptopRegularIcon,
+    StandaloneMobileNotchRegularIcon,
+} from '@deriv/quill-icons';
+import { WalletText } from '../../../../../components/Base';
+import { getDeeplinkUrl, getMobileAppInstallerUrl, getWebtraderUrl } from './constants';
+import { mobileOSDetectAsync } from './OsDetect';
+import './mt5-mobile-redirect-option.scss';
+
+type TMT5MobileRedirectOptionProps = {
+    mt5_trade_account: THooks.MT5AccountsList;
+};
+
+const MT5MobileRedirectOption: FC<TMT5MobileRedirectOptionProps> = ({ mt5_trade_account }) => {
+    const mobileURLSet = async () => {
+        window.location.replace(getDeeplinkUrl({ mt5_trade_account }));
+        const mobileAppURL = await getMobileAppInstallerUrl({ mt5_trade_account });
+        const os = await mobileOSDetectAsync();
+        const timeout = setTimeout(() => {
+            mobileAppURL && window.location.replace(mobileAppURL);
+        }, 1500);
+
+        const isSafariBrowser = () => {
+            return false;
+        };
+
+        if (!isSafariBrowser() || (os === 'iOS' && /Version\/17/.test(navigator.userAgent))) {
+            window.onblur = () => {
+                clearTimeout(timeout);
+            };
+        }
+    };
+
+    return (
+        <div className='wallets-mobile-redirect-option'>
+            <a
+                className='wallets-mobile-redirect-option__button'
+                href={getWebtraderUrl({ mt5_trade_account })}
+                target='blank'
```

Comment:
if I'm not mistaken its `target='_blank'`

Reply:
they are the same. "target='_blank'" and "target='_blank'" both serve the same purpose in HTML, which is to open a link in a new browser tab. The underscore in "_blank" is just a convention to denote a target attribute value.